src/RichText/ErrorBoundary.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 interface ErrorBoundaryProps {
 	children: React.ReactNode
-	Fallback: ({error}: {error: Error}) => React.ReactNode
+	Fallback: ({error}: { error: Error }) => React.JSX.Element
```

Comment:
So if I understand https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement correctly then:

JSX.Element (which can only be ReactElement<any, any>)

is more restrictive than

ReactNode (which can be string, number, boolean, etc)

Since Fallback is used directly as an component on line 40, your change seems correct.